packages/components/Spinner/ios/SpinnerViewManager.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+import FluentUI
+
+@objc(MSFSpinnerViewManager)
+class SpinnerViewManager: RCTViewManager {
+	override func view()->UIView! {
+		let activityIndicatorView = ActivityIndicatorView(size: .small)
+		activityIndicatorView.startAnimating()
```

Comment:
Is it by design the animation starts when it is created? because in native, the client decides when to start and stop animation.

Reply:
Also copied from fabric-react-native. Though they actually wrapped the startAnimating method in RN, and call it in "componentDidMount, which I think is like viewDidLoad

Reply:
Can remove and just let client set the prop themselves